youtube_dl/extractor/tver.py

```diff
@@ -0,0 +1,129 @@
+                'duration': 4100.032,
+                'timestamp': 1600308623,
+                'upload_date': '20200917',
+                'uploader_id': '4031511847001',
+            },
+            'skip': 'Running from test_download.py doesn\'t seem to be able to handle encrypted HLS videos',
+        },
+        {
+            'url': 'https://tver.jp/corner/f0056997',  # 'corner'
+            'md5': 'aac4e681dcdb775fc44497da4f7bdd05',  # MD5 hash of a short video downloaded by running youtube-dl with the --test option
+            'info_dict': {
+                'id': 'f0056997',  # TVer ID
+                'display_id': 'ref:kanokari_10',  # Brightcove ID
+                'ext': 'mp4',
+                'title': '彼女、お借りします　第10話「友達の彼女」-トモカノ-',
+                'description': 'バイトの初任給を何に使おうか考える和也だったが、ふと栗林のことが脳裏をよぎる。最近栗林の様子がおかしいと、木部から話を聞いていたのだ。ボーッとしていたり、女性不信のつぶやきをしているという。和也は意を決して、栗林を呼び出すことに。翌日、栗林が和也を待っていると──「駿君、だよね？」。待ち合わせ場所にやって来たのは、千鶴だった……！',
+                'thumbnail': 'https://cf-images.ap-northeast-1.prod.boltdns.net/v1/jit/5102072605001/900216cc-2e97-4c19-93bb-1a531de358d6/main/1920x1080/12m18s37ms/match/image.jpg',
+                'duration': 1476.075,
+                'timestamp': 1599554409,
+                'upload_date': '20200908',
+                'uploader_id': '5102072605001',
+            },
+            'skip': 'Running from test_download.py doesn\'t seem to be able to handle encrypted HLS videos',
+        },
+        {
+            'url': 'https://tver.jp/episode/76799350',  # 'episode'
+            'md5': 'ad893db02b8a3e949216c463af7ce51e',  # MD5 hash of a short video downloaded by running youtube-dl with the --test option
+            'info_dict': {
+                'id': '76799350',  # TVer ID
+                'display_id': '2366_2365_4533',  # Brightcove ID
+                'ext': 'mp4',
+                'title': '港時間　#49　神奈川県／リビエラシーボニアマリーナ　9月18日(金)放送分',
+                'description': '【毎週金曜 よる12時15分から放送】\n\n日本のヨット文化 を育んできた三浦半島の西海岸、小網代湾にあるリビエラシーボニアマリーナ。昨年から始まったSailGPの日本チームを率いるヨット界のレジェンドに会いました。',
+                'thumbnail': 'https://cf-images.ap-northeast-1.prod.boltdns.net/v1/jit/4394098883001/904361ca-40d3-4028-8478-8916b9a0ff49/main/1920x1080/58s80ms/match/image.jpg',
+                'duration': 116.16,
+                'timestamp': 1600052421,
+                'upload_date': '20200914',
+                'uploader_id': '4394098883001',
+            },
+            'skip': 'Running from test_download.py doesn\'t seem to be able to handle encrypted HLS videos',
+        },
+    ]
+
+    IE_NAME = 'TVer'
+    IE_DESC = 'TVer'
+
+    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(corner|episode|feature)/(?P<id>f?[0-9]+)'
+    _GEO_COUNTRIES = ['JP']  # TVer service is limited to Japan only
+
+    BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/%s/default_default/index.html?videoId=ref:%s'
+
+    def _real_extract(self, url):
+
+        # extract video id
```

Comment:
Remove all useless comments.

Reply:
Obviously excessive comments have been removed.
However, I think that the code will be difficult to understand if there are too few comments, so I left the comments that I felt necessary.